src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -141,6 +142,7 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
   useEffect(() => {
     ;(async () => {
       if (!(accountId && opportunityData?.stakedAmountCryptoBaseUnit && asset)) return
+      if (dustAmountCryptoBaseUnit && withdrawFeeCryptoBaseUnit) return
```

Comment:
This should never be hit, but I was encountering some weird re-rendering issues (possibly webpack build related), so might as well